pkg/adapter/containers.go

```diff
@@ -70,3 +72,32 @@ func (r *LocalRuntime) StopContainers(ctx context.Context, cli *cliconfig.StopVa
 	}
 	return ok, failures, nil
 }
+
+// KillContainers sends signal to container(s) based on CLI inputs.
+// Returns list of successful id(s), map of failed id(s) + error, or error not from container
+func (r *LocalRuntime) KillContainers(ctx context.Context, cli *cliconfig.KillValues) ([]string, map[string]error, error) {
+	var (
+		ok       = []string{}
+		failures = map[string]error{}
+	)
+
+	killSignal := uint(syscall.SIGTERM)
```

Comment:
This is duped between local and remote - can we move this out into cmd/podman and share it between them?

Reply:
Currently the varlink and container API use different int types.

/cc baude

Reply:
Oh. Oh dear.

Reply:
Lets make them  use the same.

Reply:
varlink currently doesn't support unsigned types while the container API uses an uint for the signal interface.  I pushed the difference down into the remote/local code.  Let me know if you like this better.